src/cryptojwt/jwe/fernet.py

```diff
@@ -0,0 +1,37 @@
+import base64
+import os
+from typing import Optional
+from typing import Union
+
+from cryptography.fernet import Fernet
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+
+from cryptojwt import as_unicode
+from cryptojwt.jwe import Encrypter
+from cryptojwt.utils import as_bytes
+
+
+class FernetEncrypter(Encrypter):
+    def __init__(self, password: str, salt: Optional[bytes] = ""):
+        Encrypter.__init__(self)
+        if not salt:
+            salt = os.urandom(16)
+        else:
+            salt = as_bytes(salt)
+
+        kdf = PBKDF2HMAC(algorithm=hashes.SHA256(), length=32, salt=salt, iterations=390000)
```

Comment:
may we get a str here, like "SHA256" and then try a getattr(hashlib, algorithm)?

Reply:
OK

Reply:
Also, use CONSTANT for 390000, e.g. `DEFAULT_PBKDF2HMAC_ITERATIONS`  or so